module/solutions/helper.js

```diff
@@ -2438,17 +2480,12 @@ module.exports = class SolutionsHelper {
 	 * @returns {Object}
 	 */
 
-	static assignedProjects(userId, search, filter, pageNo, pageSize, entityId = '') {
+	static assignedProjects(userId, search, filter, pageNo, pageSize) {
 		return new Promise(async (resolve, reject) => {
 			try {
 				let query = {
 					isDeleted: false,
-				}
-
-				if (entityId !== '') {
-					query['entityId'] = entityId // to fetch projects based on entityId key
-				} else {
-					query['userId'] = userId // to fetch projects based on userId key
+					userId: userId,
```

Comment:
@Prajwal17Tunerlabs every case we will be validating userId as well?

Reply:
yes, to deal with entityId separate code has been implemented.